yt_dlp/extractor/twitter.py

```diff
@@ -1585,6 +1587,10 @@ def _real_extract(self, url):
         if not broadcast:
             raise ExtractorError('Broadcast no longer exists', expected=True)
         info = self._parse_broadcast_data(broadcast, broadcast_id)
+        info['title'] = broadcast.get('status') or info.get('title')
+        info['uploader_id'] = broadcast.get('twitter_username') or info.get('uploader_id')
+        info['channel'] = info.get('uploader')
+        info['channel_id'] = info.get('uploader_id')
```

Comment:
IMO we should map the info_dict fields as they are in `TwitterIE`, i.e. no `channel` or `channel_id`

```suggestion
        info['uploader_url'] = format_field(broadcast, 'twitter_username', 'https://twitter.com/%s', default=None)
```